src/ViewHelpers/IfViewHelper.php

```diff
@@ -19,6 +19,16 @@
  * Alternatively, you can write a boolean expression there.
  * Boolean expressions have the following form:
  *
+ * !{variable}
+ *
+ * ::
+ *
+ *       <f:if condition="!{foo}">
+ *           Will be shown if foo is false
```

Comment:
We should be more verbose here. Is it actually `falsy`, so e.g. empty strings, `null`, etc. all evaluate to `true` when used with `!`?

Reply:
Needs to be tested. Maybe it differs for different php versions?

Reply:
True, the actual implementation is very simple:

https://github.com/TYPO3/Fluid/blob/567a16691beedbcbefa5c12f35c5e71661cfbf53/src/Core/Parser/BooleanParser.php#L353

Reply:
I see you did that by now, sounds good now.